src/components/BillFilter/BillFilter.jsx

```diff
@@ -1,227 +1,167 @@
-                      dateFilters: {
-                        ...dateFilters,
-                        startDate: {
-                          ...dateFilters.startDate,
-                          selected: false,
-                        },
-                        endDate: { ...dateFilters.endDate, selected: true },
-                      },
-                    })
-                  }
-                >
-                  End date
-                  {dateFilters.endDate.value !== ""
-                    ? `: ${dateFilters.endDate.value}`
-                    : ""}
-                </FormCheckbox>
-                <FormInput
-                  type="date"
-                  onChange={(event) => handleDateSelection(event)}
-                  disabled={
-                    dateFilters.endDate.selected === false &&
-                    dateFilters.startDate.selected === false
-                  }
-                />
-              </div>
-            )}
-            {filter.statusOpened === true && (
-              <div className="status__filter">
-                <FormCheckbox
-                  checked={billStatusFilter.resolved}
-                  onChange={() =>
-                    setState({
-                      billStatusFilter: {
-                        ...billStatusFilter,
-                        resolved: true,
-                        open: false,
-                        in_progess: false,
-                      },
-                    })
-                  }
-                >
-                  PAID
-                </FormCheckbox>
-                <FormCheckbox
-                  checked={billStatusFilter.open}
-                  onChange={() =>
-                    setState({
-                      billStatusFilter: {
-                        ...billStatusFilter,
-                        resolved: false,
-                        open: true,
-                        in_progess: false,
-                      },
-                    })
-                  }
-                >
-                  UNPAID
-                </FormCheckbox>
-                <FormCheckbox
-                  checked={billStatusFilter.in_progess}
-                  onChange={() =>
-                    setState({
-                      billStatusFilter: {
-                        ...billStatusFilter,
-                        resolved: false,
-                        open: false,
-                        in_progess: true,
-                      },
-                    })
-                  }
-                >
-                  ONGOING
-                </FormCheckbox>
-              </div>
-            )}
-            {filter.categoryOpened === true && (
-              <div className="category__filter">
-                <FormInput type="text" placeholder="Restaurant" />
-              </div>
-            )}
-          </div>
-        )}
+    <div className='bill__filter__container' ref={wrapperRef}>
+      <div className='filter__header'>
```

Comment:
is it convention to use 2 underscores

Reply:
Yes its the styling convention we decided on